.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can we copy the settings from https://github.com/GoogleChrome/chrome-extensions-samples/tree/main for consistency?